cl/search/tasks.py

```diff
@@ -2134,3 +2149,75 @@ def index_embeddings(
     if settings.ELASTICSEARCH_DSL_AUTO_REFRESH:
         # Set auto-refresh, used for testing.
         OpinionClusterDocument._index.refresh()
+
+
+@app.task(
+    bind=True,
+    max_retries=5,
+    retry_backoff=True,
+    autoretry_for=(
+        RateLimitError,
+        APIConnectionError,
+        OpenAIApiConflictError,
+        InternalServerError,
+        OpenAIApiError,
+    ),
+    retry_kwargs={"max_retries": 5},
+)
+def clean_docket_number_by_court(
+    self: Task,
+    court_batch: list[dict[int, str]],
+    court_mapping: str,
+    start_timestamp: datetime,
+    r: Redis = get_redis_interface("CACHE"),
```

Comment:
It's not advisable to use a method as default value of a celery task since it's not serializable and it's evaluated at import time.

So I suggested instead using:
```suggestion
    r: Redis | None= None,
```

As I noticed you're passing the redis interface in tests.

So within the task you can instead do:

```
   if r is None:
        r = get_redis_interface("CACHE")
```